src/database-sqlite3.cpp

```diff
@@ -162,14 +180,53 @@ void Database_SQLite3::openDatabase()
 	SQLOK(sqlite3_busy_handler(m_database, Database_SQLite3::busyHandler,
 		m_busy_handler_data), "Failed to set SQLite3 busy handler");
 
+	bool created = false;
 	if (needs_create) {
+		created = true;
```

Comment:
Instead of adding this new `created` variable you just just use the existing `needs_create` variable.